### PR TITLE
Pydeck EarthEngineTerrainLayer

### LINKED ISSUE #90: EarthEngineTerrainLayer for Python
Would  `EarthEngineTerrainLayer` be available for `pydeck_earthengine_layers` in the near future?  I tried to replicate the [terrain example](https://earthengine-layers.com/examples/terrain) using Jupyter notebook, but it seems the  `EarthEngineTerrainLayer` class is not yet available for Python. Thanks.

### PR DESCRIPTION
**Pull request by kylebarron**
Closes #90

Adds the ability to call the EarthEngineTerrainLayer from Python.

- Updates `rollup/plugin-commonjs` to fix some bundling issues with dependencies
- Removes `@luma.gl` from rollup externals. Since we never call luma ourselves anywhere in EarthEngine layer, declaring it as external hid the bundled copy of luma.gl within `deck.gl/geo-layers`
- Adds more `@deck.gl` externals. Full list derived from `jupyter-widget` [here](https://github.com/visgl/deck.gl/blob/c8702ae134e0598b2fdca03642744f25e9de593b/modules/jupyter-widget/src/deck-bundle.js). Otherwise had multiple copies of deck.gl bundled, which caused "Model needs a Program" errors
- [x] Notebook example of the `EarthEngineTerrainLayer`

Questions:

- [x] what externals should be declared for the non-Pydeck EE bundle? Should we assume that all deck.gl modules are loaded separately?

### CODE REVIEW

File: modules/earthengine-layers/package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "@math.gl/web-mercator": "3.1.2"
   },
   "devDependencies": {
-    "@rollup/plugin-commonjs": "^11.0.2",
+    "@rollup/plugin-commonjs": "^15.0.0",
```
Comment (ibgreen) [inline modules/earthengine-layers/package.json:25]: Should we also bump earthengine api module to latest?

Comment (ibgreen) [reply modules/earthengine-layers/package.json:25]: If so, maybe better as separate PR.

Comment (kylebarron) [reply modules/earthengine-layers/package.json:25]: Yeah probably good for a different PR

**Comment by giswqs** [comment]
Excellent work! Thank you very much for implementing this. I will test it shortly.

**Pull request merged**